wrapanapi/systems/openstack.py

```diff
@@ -584,7 +585,15 @@ class OpenstackSystem(System, VmMixin, TemplateMixin):
     can_pause = True
 
     def __init__(self, **kwargs):
-        self.keystone_version = kwargs.get('keystone_version', 2)
+        self.keystone_version = kwargs.get("keystone_version")
```

Comment:
**Required** would like to the required args explicitly defined here in the init instead of just **kwargs. This is how the rest of the system init method are done.